clippy_config/src/conf.rs

```diff
@@ -471,6 +471,9 @@ define_Conf! {
     /// A list of paths to types that should be treated as if they do not contain interior mutability
     #[lints(borrow_interior_mutable_const, declare_interior_mutable_const, ifs_same_cond, mutable_key_type)]
     ignore_interior_mutability: Vec<String> = Vec::from(["bytes::Bytes".into()]),
+    /// The maximum size of a `T` in `RefCell<T>` to suggest to swap to `Cell` if applicable.
+    #[lints(copy_refcell)]
+    large_cell_limit: u64 = 256,
```

Comment:
Nit: Is there a specific reason for the limit? There is prior art for 128 [here](https://github.com/rust-lang/rust-clippy/issues/4652#issue-505670554), also used [here](https://github.com/rust-lang/rust-clippy/pull/9373). In any case, since comparison is done as `>`, it probably should be 255, not 256.

Reply:
I just randomly picked a sensible number, I didn't know there was prior art, I'll change to 128 and fix the comparison.